idl/floatingBaseEstimatorRPC/autogenerated/codyco_floatingBaseEstimatorRPC_thrift.cmake

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2006-2018 Istituto Italiano di Tecnologia (IIT)
+# All rights reserved.
+#
+# This software may be modified and distributed under the terms of the
+# BSD-3-Clause license. See the accompanying LICENSE file for details.
+
+## This is an automatically-generated file.
+## It could get re-generated if the ALLOW_IDL_GENERATION flag is on
+
+set(headers include/codyco/HomTransform.h;include/codyco/floatingBaseEstimatorRPC.h)
+set(sources src/HomTransform.cpp;src/floatingBaseEstimatorRPC.cpp)
```

Comment:
Is it safe to also copy autogenerated files?

Reply:
Again, they were committed also in the old repo, and they worked  fine. If we want to improve this, let's do this in a following PR.